README.md

```diff
@@ -143,49 +119,65 @@ Run the container in interactive mode with your favorite docker flags, for examp
 $ docker run --name my_container_name -it -v /my/host/volume/:/mounted/volume --device=/dev/dri my_tag
 ```
 
-where `my_tag` would be `jjerphan/numba_dpex_dev:latest` if you pulled from the repository.
+where `my_tag` would be `jjerphan/numba_dpex_dev:latest` if you pulled from the
+repository.
 
-⚠ The flag `--device=/dev/dri` is **mandatory** to enable the gpu within the container, also the user starting the `docker run` command must have access to the gpu, e.g. by being a member of the `render` group.
+⚠ The flag `--device=/dev/dri` is **mandatory** to enable the gpu within the container,
+also the user starting the `docker run` command must have access to the gpu, see
+[Give permissions to submit GPU workloads](#give-permissions-to-submit-gpu-workloads).
 
-Unless using the flag `--rm` when starting a container, you can restart it after it was exited, with the command:
+Unless using the flag `--rm` when starting a container, you can restart it after it was
+exited, with the command:
 
 ```
-sudo docker start -a -i my_container_name
+$ sudo docker start -a -i my_container_name
 ```
 
-Once inside the container, you can check that the environment works: the command
+### Step 2: Check the installation of the environment was successfull
+
+Once inside the environment you just installed with one of those two methods, you can
+check that the environment works by introspecting the available hardware:
 
 ```
-python -c "import dpctl; print(dpctl.get_devices())"
+$ python -c "import dpctl; print(dpctl.get_devices())"
 ```
 
-will introspect the available hardware, and should display working `opencl` cpu and gpu devices, and `level_zero` gpu devices.
+this should print a list of available devices, including `cpu` and `gpu` devices, once
+for each available backends (`opencl`, `level_zero`,...).
 
-### Step 2: install the `wip-engines` branch of scikit-learn
+### Step 3: install the `wip-engines` branch of scikit-learn
 
-Once you have loaded into a `numba_dpex` development environment, following one of the two previous guides, follow those instructions:
+TODO: rather than expecting user to manually install this branch, release an end-to-end
+conda build.
+
+Once you have loaded into a `numba_dpex` environment, follow those instructions:
 
 ```
 git clone https://github.com/ogrisel/scikit-learn
 cd scikit-learn
 git checkout wip-engines
-pip install -e . --no-build-isolation
+pip install -e .
 cd ..
 ```
 
-### Step 3: install this plugin
+### Step 4: install this plugin
+
+TODO: rather than expecting user to manually install this branch, release an end-to-end
+conda build.
+
+FIXME: currently, non-editable mode installation does not work.
```

Comment:
Should we create an issue for this?

Reply:
You can but it's on my personal list of tasks anyway, if we want to create a conda build we must create a wheel and that means fixing that first.